custom_components/bosch_alarm/device.py

```diff
@@ -2,11 +2,30 @@
 
 from .const import DOMAIN
 
-def device_info_from_panel(panel):
-    return DeviceInfo(
-        identifiers={(DOMAIN, panel.serial_number)},
-        name=f"Bosch {panel.model}",
-        manufacturer="Bosch Security Systems",
-        model=panel.model,
-        sw_version=panel.firmware_version,
-    )
+class PanelConnection:
+    def __init__(self, panel, unique_id, model):
+        self.unique_id = unique_id
+        self.model = model
+        self.panel = panel
```

Comment:
nit: please keep the order of the initilizers the same as the order of the arguments, where possible